stdlib/Profile/src/Profile.jl

```diff
@@ -1193,17 +1193,17 @@ end
 
 # Utilities
 function rtruncto(str::String, w::Int)
-    if length(str) <= w
+    if textwidth(str) <= w
         return str
     else
-        return string("...", str[prevind(str, end, w-4):end])
+        return string("…", str[prevind(str, end, w-2):end])
```

Comment:
This won't work right with composites, will it? But of course that's an existing issue with the code; and one might argue that usually it won't matter in this context. Unless someone heavily uses accents in function names perhaps?

I.e., compare this:
```
julia> s = "áb"  # input via ""a\acute<TAB>b""
"áb"

julia> collect(s)
3-element Vector{Char}:
 'a': ASCII/Unicode U+0061 (category Ll: Letter, lowercase)
 '́': Unicode U+0301 (category Mn: Mark, nonspacing)
 'b': ASCII/Unicode U+0062 (category Ll: Letter, lowercase)

julia> using Unicode

julia> collect(Unicode.graphemes(s))
2-element Vector{SubString{String}}:
 "á"
 "b"
```


Anyway, don't take this as an objection to this PR from me -- this issue will never affect me, I only ever use ASCII function names, so I don't mind either way ;-)